src/components/pages/SideDrawer/SideDrawer.js

```diff
@@ -46,7 +46,11 @@ function SideDrawer() {
           )}
         </Button>
 
-        <Menu.Item className="cases" key="/">
+        <Menu.Item className="home" key="/">
```

Comment:
This is great - I've already built the routing out for the new component, so this should match perfectly when I merge that code in later today!